### ISSUE TITLE
Support for TypeScript 5.x

### ISSUE THREAD

**Issue by JapuDCret** [dilanx/craco#503]
Dependency to CRA should be updated, once https://github.com/facebook/create-react-app/issues/13080 is resolved

**Comment by dilanx** [comment]
Yeah, we can follow CRA's dependencies, but who knows when that'll be resolved 😭
